openedx/core/djangoapps/credit/models.py

```diff
@@ -18,6 +18,16 @@
 log = logging.getLogger(__name__)
 
 
+class CreditProvider(TimeStampedModel):
+    """
+    This model represents an institution that can grant credit for a course.
+    Each provider is identified by unique ID (e.g., 'ASU').
+    """
+
+    provider_id = models.CharField(max_length=255, db_index=True, unique=True)
```

Comment:
I hear the concern, but I'm inclined to keep it as `provider_id` to avoid introducing more conflicts with the other branches in-flight (we're all working off a design spec that calls it `provider_id`)